src/InputWithLabel.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useRef, useEffect } from 'react';
```

Comment:
Nice touch of making this component more abstract then what the directions ask for. This is a great design pattern when making a reusable modular component!